cpp/examples/jsonschema/writer.cpp

```diff
@@ -39,8 +39,8 @@ int main(int argc, char** argv) {
 
   mcap::McapWriter writer;
   {
-    auto options = mcap::McapWriterOptions("x-jsonschema");
-    const auto res = writer.open(outputFilename, options);
+    auto options = mcap::McapWriterOptions("");
+    const auto res = writer.open(outputFilename);
```

Comment:
is there something that is still illustrative we can put instead of these empty strings? otherwise as a reader it's unclear why the API even needs these strings

Reply:
What would be a sensical value to put in here?

> The profile is used for indicating requirements for fields throughout the file (encoding, user_data, etc). If the value matches one of the [well-known profiles](https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-profiles), the file should conform to the profile. This field may also be supplied empty or containing a framework that is not one of those recognized.

Since this is just an example file I'm not sure there are any requirements we care to specify. And it's not really being written by a framework (although I thought the framework info is what would go in `library`, so not sure why we mention frameworks here)